qiskit/result/mitigation/correlated_readout_mitigator.py

```diff
@@ -263,6 +265,6 @@ def stddev_upper_bound(self, shots: int):
         return gamma / np.sqrt(shots)
 
     @property
-    def qubits(self) -> Tuple[int]:
+    def qubits(self) -> tuple[int]:
         """The device qubits for this mitigator"""
-        return self._qubits
+        return self._qubits  # TODO: cast to tuple or change return type
```

Comment:
This appears to be just the wrong signature. `self._qubits` is not necessarily tuple.